src/Microsoft.ML.Data/DataLoadSave/Binary/CodecFactory.cs

```diff
@@ -54,11 +54,11 @@ public CodecFactory(IHostEnvironment env, MemoryStreamPool memPool = null)
             RegisterSimpleCodec(new UnsafeTypeCodec<ulong>(this));
             RegisterSimpleCodec(new UnsafeTypeCodec<Single>(this));
             RegisterSimpleCodec(new UnsafeTypeCodec<Double>(this));
-            RegisterSimpleCodec(new UnsafeTypeCodec<DvTimeSpan>(this));
+            RegisterSimpleCodec(new UnsafeTypeCodec<TimeSpan>(this));
```

Comment:
>UnsafeTypeCodec [](start = 36, length = 15)

Is it still need to be UnsafeTypeCodec?

Reply:
What should it be? I have defined TimeSpanUnsafeTypeOps to handle this type.

---
In reply to: [211110052](https://github.com/dotnet/machinelearning/pull/693#discussion_r211110052) [](ancestors = 211110052)